### ISSUE TITLE
how to define my own loss function in keras

### ISSUE THREAD

**Issue by kingfengji** [fchollet/keras#2066]
Hi, how can I define my own objective function in keras?
THanks!

**Comment by kingfengji** [comment]
yeah I know how. now.

**Comment by Sandy4321** [comment]
So what is the answer

**Comment by alyato** [comment]
@kingfengji Do you share one example?
